.agents/WAVE4-TEST-IMPROVEMENT-PLAN.md

```
@@ -0,0 +1,518 @@
+
+Expected breakdown:
+
+- Qwiq.Client.Rest: 0% → 60%
+- Qwiq.Core: 51.2% → 60%
+- Overall: 51.1% → 65%
+
+---
+
+## Phase 3: Coverage Expansion - Remaining Gaps (Weeks 7-8)
+
+**Goal**: Close remaining gaps to achieve 70% target
```

markdownlint-cli2 (0.18.1)

Emphasis used instead of a heading

(MD036, no-emphasis-as-heading)